FF116 Audio Output Device API updates (#27818)
mdn/content: files/en-us/web/api/mediadevices/enumeratedevices/index.md

{{APIRef("WebRTC")}}

The {{domxref("MediaDevices")}} method **`enumerateDevices()`** requests a list of the currently available media input and output devices, such as microphones, cameras, headsets, and so forth.

> The returned {{jsxref("Promise")}} is resolved with a {{domxref("MediaDeviceInfo")}} array describing the devices.

Reviewer: ```suggestion
The returned {{jsxref("Promise")}} is resolved with an array of {{domxref("MediaDeviceInfo")}} objects describing the devices.
```

### Return value

> A {{ jsxref("Promise") }} that receives an array of {{domxref("MediaDeviceInfo")}} objects when the promise is fulfilled.

Reviewer: ```suggestion
A {{ jsxref("Promise") }} that is fulfilled with an array of {{domxref("MediaDeviceInfo")}} objects.
```

(I think we usually say this rather than "receives")
